lib/assets/i18n/log/messages_ru.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Ignore the following for cluster support testing 

(This is testing for the logger messages substitution due to my unclean branch)